collectors/PropertiesCollector.py

```diff
@@ -35,8 +39,8 @@ def collect(self):
             logger.warning(f'No values in the response for {self.name}. API code: {api_responding}')
             return
 
-        no_match_in_config = set()
         values_received = set()
+        no_match_in_config = []
```

Comment:
```suggestion
        no_match_in_config = list()
```

Meanwhile I'd prefer this format :)